README.md

```diff
@@ -25,4 +25,11 @@ client.mailbox_get({
     accountId: Object.keys(jmapClient.getSession().accounts)[0],
     ids: null,
 })
-```
\ No newline at end of file
+```
+
+Push (web sockets) usage example for mailboxes
+```typescript
+const mailboxObservable = client.pushMailbox().subscribe();
+client.pushStart();
```

Comment:
Subscribe and start are similar in semantic. I d expect only one.

Where is the callback?

Also it would imo be nice to support push notifications for arbitrary types...

Reply:
> Subscribe and start are similar in semantic. I d expect only one.

Calling pushStart then subscribe is not the same than calling subscribe then pushStart.
We can talk about a way you would like to call methods, a scenario, and what you expect to happen under the hood.

> Where is the callback?

You can use a callback as parameter of the subscribe method, or you can do what you want with the mailboxObservable. See https://rxjs.dev/guide/overview

> Also it would imo be nice to support push notifications for arbitrary types...

What would be the use case?